echo-notifications/src/main/groovy/com/netflix/spinnaker/echo/notification/EmailNotificationAgent.groovy

```diff
@@ -75,6 +74,8 @@ class EmailNotificationAgent extends AbstractEventNotificationAgent {
       status == 'complete' ? 'completed successfully' : status
     }"""
 
+    subject = context.customSubject ?: subject
```

Comment:
does the [Spinnaker] gets lost here before customSubject? is that intentional?

Reply:
The [Spinnaker] was always getting lost. #L61-75 is an `if/else if/else` block, and `subject` is set in each one.

Reply:
makes sense